index.html

```diff
@@ -171,46 +178,40 @@ <h2 class="projectname about">Frameworks</h2>
                         <div class="projectdetail about">
                             <h2 class="projectname about">Skills</h2>
-                        <ul class="skills about">
-                            <li>GitHub</li>
-                            <li>CodePen</li>
-                            <li>GitLab</li>
-                            <li>Terminal</li>
+                            <ul class="skills about">
+                                <li>GitHub</li>
+                                <li>CodePen</li>
+                                <li>GitLab</li>
+                                <li>Terminal</li>
 
-                        </ul>
-                    </div>
+                            </ul>
+                        </div>
                     </div>
                 </li>
 
             </ul>
         </div>
 
     </section>
-    <section>
-        <div class="container form">
-        <h2 class="cta">I'm always interested
-             in hearing about new
-              projects, so if you'd
-               like to chat please
-                get in touch.</h2>
-            </div>
-            <form action="https://formspree.io/f/mknerreb" method="POST">
-                <div class="container form">
-                    <div class="form-group">
-                        <input type="text" name="name" id="name" placeholder="Full name" required>
-                        <input type="email" name="email" id="email" placeholder="Email address" required>
-                        <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" required></textarea>
-                        <button type="submit" class="projectbutton">Get in touch</button>
-                    </div>
-                </div>
-                <ul class="social">
-                    <li><a href="https://twitter.com/LukmanAbdulka18"><img src="images\twitter.svg" alt="twitter"></a></li>
-                    <li><a href="#"><img src="images\Linkedin.svg" alt="linkedin"></a></li>
-                    <li><a href="#"><img src="images\medium.svg" alt="medium"></a></li>
-                    <li><a href="https://github.com/lukman155"><img src="images\github.svg" alt="github"></a></li>
-                    <li><a href="#"><img src="images\angellist.svg"></a></li>
-                </ul>
-                <h3>lukmanabdulkarim15@gmail.com</h3>
+    <section class="contact-us container">
+        <p class="contact">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+            touch.</p>
+        <form action="https://formspree.io/f/mknerreb" method="POST" class="get-in-touch">
+            <input type="text" placeholder="Full name" id="name" maxlength="30" name="name" required>
+            <input type="email" name="email" id="email" placeholder="Email address" required>
+            <textarea name="message" id="message" placeholder="Enter text here" cols="30" rows="10" required></textarea>
```

Comment:
- [ ]  Kindly add an attribute `maxlength="500"` to your `textarea` as per requirement 🤓 

![Captura de Pantalla 2022-09-27 a la(s) 8 46 15 a  m](https://user-images.githubusercontent.com/86022690/192543859-05d7f3f0-4019-48cd-bf22-08e5c268118c.png)